main.py

```diff
@@ -2517,8 +2521,11 @@ async def gen_inventory(message, person_id):
             give_collector = False
 
     if user.custom:
+        custom_num = 1
+        if user.custom_num:
+            custom_num = user.custom_num
         icon = get_emoji(user.custom.lower().replace(" ", "") + "cat")
-        embedVar.add_field(name=f"{icon} {user.custom}", value=1, inline=True)
+        embedVar.add_field(name=f"{icon} {user.custom}", value=custom_num, inline=True)
```

Comment:
just do
```suggestion
        embedVar.add_field(name=f"{icon} {user.custom}", value=user.custom_num, inline=True)
```
and remove the 3 new lines above